cli/args/flags.rs

```diff
@@ -1810,6 +1824,15 @@ fn permission_args(app: Command) -> Command {
           Ok(())
         }),
     )
+    .arg(
+      Arg::new("allow-sys-info")
+        .long("allow-sys-info")
+        .min_values(0)
+        .takes_value(true)
+        .use_value_delimiter(true)
+        .require_equals(true)
+        .help("Allow access to system info"),
```

Comment:
Right now this flag can accept arbitrary values, I suggest to add a validator here that will limit options

Reply:
added validation of sys kind in 3 places:
- `allow-sys` arg values are validated
- `kind` prop of `SysPermissionDescriptor` is now typed as `"hostname" | "loadavg" | ...` (in TypeScript)
- `op_{query|revoke|request}_permission` throws TypeError when the kind is invalid